string emotion
float32 probability